content/doc/book/system-administration/administering-jenkins.adoc

```diff
@@ -0,0 +1,101 @@
+---
+layout: section
+title: Administering Jenkins
+---
+
+== `JENKINS_HOME` directory
+Jenkins needs some disk space to perform builds and keep archives.
+You can check this location from the configuration screen of Jenkins.
+By default, this is set to `~/.jenkins`, but you can change this in one of the following ways:
+
+* Set the `JENKINS_HOME` environment variable to the new home directory before launching the servlet container.
+* Set the `JENKINS_HOME` system property in the servlet container.
+* Set the `JNDI` environment entry `JENKINS_HOME` to the new directory.
+See the link:/doc/book/installing/index[Installing Jenkins] for more about how to do this for your container.
+
+You can change this location after you've used Jenkins for a while, too.
+To do this, stop Jenkins completely, move the contents from the old `JENKINS_HOME` to the new home, set the new `JENKINS_HOME`, and restart Jenkins.
```

Comment:
```suggestion
To do this:

1. Stop Jenkins completely
1. Move the contents of the old `JENKINS_HOME` to the new location
1. Set the `JENKINS_HOME` variable to the new location
1. Restart Jenkins.
```
I think this warrants a list of steps that are easier for people to see than just the solid text